### PR TITLE
chore: [#185084111] Add a locked tool tip to tax

### PR DESCRIPTION
**Pull request by cbrunefearless**
### Ticket

[#185084111](https://www.pivotaltracker.com/story/show/)

### Approach

Added tax masking on register for taxes

## Code author checklist

- [X] I have performed a self-review of my code
- [X] I have created and/or updated relevant documentation, if necessary
- [X] I have not used any relative imports
- [X] I have checked for and removed instances of unused config from CMS
- [X] I have pruned any instances of unused code
- [X] I have not added any markdown to labels, titles and button text in config

### CODE REVIEW

**Review by jphechter — APPROVED** [review]

File: web/src/components/tasks/TaxInput.tsx [hunk 1]
```diff
@@ -85,6 +85,9 @@ export const TaxInput = (props: Props): ReactElement => {
       .update()
       .then(() => {
         setIsLoading(false);
+        updateQueue
+          .queueTaxFilingData({ ...taxFilingData, state: "SUCCESS", registeredISO: undefined, filings: [] })
+          .update();
```
Comment (somabadri) [inline web/src/components/tasks/TaxInput.tsx:90]: Why is the tax filing data here being explicitly set to `SUCCESS`? We are not hitting the API to confirm that the tax filing state is successful right?

Comment (cbrunefearless) [reply web/src/components/tasks/TaxInput.tsx:90]: Should we change it to Pending?  Or should we change the logic for it?

Comment (cbrunefearless) [reply web/src/components/tasks/TaxInput.tsx]: Changed!

File: web/src/components/onboarding/taxId/DisabledTaxId.tsx [hunk 2]
```diff
@@ -110,8 +110,16 @@ export const DisabledTaxId = (props: Props): ReactElement => {
...
-      <span className={taxIdDisplayStatus === "password-view" ? "text-ls-3" : ""}>{value.slice(0, 9)}</span>
-      <span>{value.slice(9)}</span>
+      {taxIdDisplayStatus === "password-view" ? (
+        <span data-testid={"disabled-tax-id-value"} className={"password-view"}>
+          {"****-****-****"}
+        </span>
+      ) : (
+        <>
+          <span data-testid={"disabled-tax-id-value"}>{value.slice(0, 9)}</span>
+          <span>{value.slice(9)}</span>
```
Comment (somabadri) [inline web/src/components/onboarding/taxId/DisabledTaxId.tsx:120]: If we are just displaying it in plain text, I think we can change it just being in just one span right? we don't need to slice it anymore?

Comment (cbrunefearless) [reply web/src/components/onboarding/taxId/DisabledTaxId.tsx]: correct and done

**Pull request merged**